core: add command to check cluster health in dr (#57)
rook/kubectl-rook-ceph: docs/dr-health.md

> # Disaster Recovery

Reviewer: @netzzer Any more suggestions on this usage or the expected output.

> 

Reviewer: The Health header needs to be before this paragraph
```suggestion
## Health
```

> DR health command is used to get connection status of one cluster from another cluster in mirroring enabled clusters. We fetch cephblockpool with mirrroing enabled if not found it will exit with relevant logs. Optionally, it accept all the args that are supported with `ceph status` command like `--debug-ms 1`.

Reviewer: This paragraph and example needs to be moved inside the Health header.
```suggestion
The DR health command is used to get the connection status of one cluster from another cluster in mirroring-enabled clusters. The cephblockpool is queried with mirrroring enabled and If not found will exit with relevant logs. Optionally, args can be passed to be appended to `ceph status`, for example: `--debug-ms 1`.
```

Author: changed

Example: `kubectl rook-ceph dr health [ceph status args]`

> ## Command

Reviewer: In the future there may be multiple DR commands, so we can have one section for each. The above paragraph and example would move inside this header.
```suggestion
## Health
```

Author: Agree!
